### ISSUE TITLE
Roku Harmony Span device is not a home control device

### ISSUE THREAD

**Issue by xpouyat** [AShifter/harmony-span#20]
Thanks for the software. It works great.
I tried to assign the home control buttons of the Harmony Elite when NO activities is launched. The logitech app refuses and says "Home control devices can only be assigned to these buttons. You have not added home control devices".

To get this message, follow
https://support.myharmony.com/en-us/customizing-physical-buttons-on-your-remote
but select "Home Control Buttons" instead of "Activity Control Buttons".

So while I can control lights when an activity is launched, I cannot control lights when no activity is launched. This is because the Roku Span device is not a home control device.

**Comment by michaelewers** [comment]
Hi,

you need a tool that "fakes" either of the supported Smart Home systems, like [diyHue](https://github.com/diyhue/diyHue). But I'm not sure if that works without the Raspbee module installed.

M.

**Comment by AShifter** [comment]
@xpouyat Yes, this is expected behavior right now. the diyHue project that @michaelewers may be a suitable solution, but this likely won't be a feature added to HarmonySpan itself. Let us know what kind of success you have with this!

**Comment by xpouyat** [comment]
Thanks. I don't think that I will investigate diyHue as I don't want to deploy an extra hardware for now. To workaround the limitation, I created a "empty" activity just to control the lights.
